machine-learning/README.md

```diff
@@ -1,4 +1,99 @@
+
+Anything where normally we use our intution to predict a pattern, ML can be applied to do the work quicker and better!
+
+## Example Projects
+Here are a couple of the cool ML hackathon projects as some inspiration to get started!
+
+1. [NavAssistAI](https://devpost.com/software/navassistai) - This project won the Tensorflow World 2.0 Challenge, and attempts to help visually impaired individuals cross the street using machine learning. I like this project because its very applicable and usable in real life. 
+
+2. [Coughvid](https://devpost.com/software/coughvid-ai-based-app-for-covid-19-screening-using-coughing) - This project placed 3rd at LauzHack for COVID. It uses ML to analyze coughing sounds and symptoms to predict if a patient has COVID-19.
+
+3. [ColorMe](https://devpost.com/software/colorme-50fz68) - A unique project which usual Image Analysis to predict what color palettes for clothing would look best on the user.
+
+4. [StickyAR](https://devpost.com/software/stickyar) - Fun game built at HackMIT that takes a more light-hearted approach to ML. These hackers used OpenCV to detect sticky notes in a frame, and let the end user play a game using those sticky notes!
+
+There are a lot more! Browse [devpost](https://devpost.com) to see more cool projects. 
+
+## Tools and libraries to get you started
+If you're gonna do any kind of ML project, I'd highly recommend Python. Many common Python ML libraries such as Keras, PyTorch, and Tensorflow abstract alot of the process for you, so you can focus on the bigger picture! This is especially good for beginners who may not understand everything that is going on behind the scenes just yet. C++ and Java are also pretty popular options, but I can't speak to their usability.
+
+Keras, PyTorch, and Tensorflow have a whole bunch of documentation out there for how to set them up, and how to use them. I'll link a couple python tutorials I found really useful down below. The best part about this library is that it works right out of the box (for the most part), allowing you to quickly move right on integrating it with the rest of the project.
+
+You will most likely have to look into these frameworks to decide which one is the right fit for your project, but here are some general differences as a starting point:
+
+Keras (Recommended for beginners and small datasets):
+  - High level API (Abstracts things more)
+  - Slower runtimes(Shouldn't matter too much in this context)
+  - Simplest to use
+  - Least lines of code
+  - Less accurate models
+  - Simple models with minial debugging  
+  
+Tensorflow (Recommended for experienced users and high performance reduced size models):
```

Comment:
maybe explain what you mean by "high performance reduced size models", even i'm not sure what that means